src/components/chat/unlockProfile/UnlockProfile.tsx

```diff
@@ -82,7 +82,7 @@ const UnlockProfile = ({ InnerComponentProps, onClose }: UnlockProfileModalProps
       setActiveStatus({
         status: PROFILESTATE.UNLOCK_PROFILE,
         title: 'Unlock Profile',
-        body: description ? description : 'Unlock your profile to read and send messages.',
+        body: description ? description : 'Unlock your profile to proceed.',
```

Comment:
@mishramonalisha76 we will not change the text in this component as this will affect other places as well. We can pass the description as a prop from outside.